www/main/launchpad/Launchpad.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useStoreState, useStoreActions } from 'easy-peasy';
+
+import Toolbar from '../components/toolbar';
+import Navigator from '../components/navigator';
+
+import { Ceremonies, SundarGutka, BaniController, LockScreen } from '../components/addons';
+
+import { DEFAULT_OVERLAY } from '../constants';
+
+const Launchpad = () => {
+  const { overlayScreen } = useStoreState(state => state.app);
+  const { setOverlayScreen } = useStoreActions(actions => actions.app);
+
+  const onScreenClose = React.useCallback(() => {
+    setOverlayScreen(DEFAULT_OVERLAY);
+  }, [setOverlayScreen, DEFAULT_OVERLAY]);
+
+  const isSundarGutkaOverlay = overlayScreen === 'sunder-gutka';
+  const isBaniControllerOverlay = overlayScreen === 'sync-button';
+  const isCeremoniesOverlay = overlayScreen === 'ceremonies';
+  const isLockScreen = overlayScreen === 'lock-screen';
+
+  return (
+    <div className="launchpad">
+      <Toolbar />
+      {isSundarGutkaOverlay && <SundarGutka onScreenClose={onScreenClose} />}
```

Comment:
feels like duplication, would a switch statement here been simpler?

Reply:
Umm, not sure. Hehe! 

I can do switch if you suggest that, I found this to be more cleaner code.